workflow-templates/phpunit-mysql.yml

```diff
@@ -7,6 +7,18 @@ name: PHPUnit
 
 on:
   pull_request:
+    paths:
```

Comment:
Used since a long time in spreed/talk
https://github.com/nextcloud/spreed/blob/c142394f0e18e7b788437eea5868dc7555d6d73a/.github/workflows/phpunit.yml#L5